lib/engine/game/g_18_carolinas/game.rb

```diff
@@ -297,14 +343,62 @@ def operating_round(round_num)
           Round::Operating.new(self, [
             Engine::Step::Bankrupt,
             Engine::Step::HomeToken,
-            Engine::Step::Track,
+            G18Carolinas::Step::Track,
             Engine::Step::Token,
             Engine::Step::Route,
             Engine::Step::Dividend,
             Engine::Step::BuyTrain,
           ], round_num: round_num)
         end
 
+        def upgrades_to?(from, to, special = false, selected_company: nil)
+          standard = to.paths.any? { |p| p.track == :broad }
+          southern = to.paths.any? { |p| p.track != :broad }
+
+          north = @north_hexes.include?(from.hex)
+          south = @south_hexes.include?(from.hex)
```

Comment:
do you really need both? aren't they opposites?

Reply:
No. There are some hexes with both.